CHANGELOG.md

```diff
@@ -23,10 +23,12 @@ These changes likely break some apps, please read the
 
 * Change bundled resources to use more efficient string storage
 * Desktop left and right mouse buttons renamed to MouseButtonPrimary and MouseButtonSecondary
+* Many optimisations and widget performance enhancements
 
 ### Fixed
 
-*
+* Validating a widget in widget.Form before renderer was created could cause a panic
+* Appended items in widget.Form had validation set up twice
```

Comment:
This line seems like a duplicate of line 26 (the widget optiomisations comment)
Is the line above a bug that developers could encounter, or was it just our internal code?

Reply:
Good point. Changed it.